pontoon/base/static/js/lib/fluent-syntax.js

```diff
@@ -1,15 +1,31 @@
-/* fluent-syntax@0.3.0 */
+/* fluent-syntax@0.4.0 */
```

Comment:
Ok, I changed `fluent_interface.js`. From my small QA it looks like everything works, but I don't feel confident if my clicking scanned through all codepaths.